apollo/helpers.ts

```diff
@@ -45,7 +46,17 @@ export const getAllUserIds = (): string[] => {
 };
 
 export const getTalentById = async (id: string): Promise<Talent> => {
-  return await models.Talent.findOne({ id });
+  let talent: Talent | null = null;
+  try {
+    talent = await models.Talent.findOne({ id });
+  } catch (e) {
+    handleError(e);
+  }
+  if (!talent)
```

Comment:
Does findOne return "undefined" if it didn't find a talent?
If yes, then this check works.

Reply:
I think it returns null, which is also falsy. It works, too: try accessing a url which does not correspond with an existing id!